### PR TITLE
Fixed incorrect function call of normalize_data_format with keras update

### PR DESCRIPTION
**Pull request by udayakumar97**
With the keras update, normalize_data_format is no longer in conv_utils. Fixed this by importing normalize_data_format from keras.backend

### CODE REVIEW

**Comment by bonlime** [comment]
Thanks for making a pull request!
